### PR TITLE
Pull recent updates from nessi.no-2022.11-dev into PR39

### PR DESCRIPTION
**Pull request by trz42**

### CODE REVIEW

File: eessi-2022.11.yml [hunk 1]
```diff
@@ -2,6 +2,7 @@ easyconfigs:
   - EasyBuild-4.7.0.eb
   - CMake-3.20.1-GCCcore-10.3.0.eb
   - Python-3.9.5-GCCcore-10.3.0.eb
+  - Qt5-5.14.1-GCCcore-9.3.0.eb
```
Comment (TopRichard) [inline eessi-2022.11.yml]: Please change the order to match the ingestion seq

**Review by TopRichard — CHANGES_REQUESTED** [review]
Mostly fine but check the comment

File: eessi-2022.11.yml (hunk 1 above)
Comment (trz42) [reply eessi-2022.11.yml]: Done in https://github.com/TopRichard/bot-software-layer1/pull/11/commits/a3dd3eb3ef04f6233bb0b79acf3755aea8db7b26

**Review by TopRichard — APPROVED** [review]
Fine

**Pull request merged**